tests/backup/backup_helper.go

```diff
@@ -3447,3 +3448,171 @@ func GetVolumeMounts(AppContextsMapping *scheduler.Context) ([]string, error) {
 	}
 	return nil, fmt.Errorf("unable to find the mount point for %s", AppContextsMapping.App.Key)
 }
+
+// UpdateSchedulePolicyOwnership Updates the schedulePolicy object ownership
+func UpdateSchedulePolicyOwnership(schedulePolicyName string, schedulePolicyUid string, userNames []string, groups []string, accessType api.Ownership_AccessType, ctx context.Context) error {
```

Comment:
Need to handle case where you can provide access to everyone. With this you need to pass user & group.

Reply:
Added publicAccess argument.
Thanks @venkatpx .